Modules/@babylonjs/react-native-basekit/README.md

```diff
@@ -0,0 +1,120 @@
+# Babylon React Native Basekit
+
+## Usage
+
+This quick overview will help you understand the constructs provided by Babylon React Native and how to use them in a React Native application.
+
+### Dependencies
+
+This package has several **peer dependencies**. If these dependencies are unmet, `npm install` will emit warnings. Be sure to add these dependencies to your project.
+
+### Installation Configuration
+
+Downloaded dependencies are cache by default in directory specifiec by `npm_config_cache`. It's possible to specify a new cache directory by setting `npm_config_babylon_binary_cache` environment variable.
+
+Also, it's possible to set the source of these downloaded dependencies. By default, it's using main Github repo releases: `https://github.com/BabylonJS/BabylonReactNative/releases/download`.
+Other possibilities are:
+- By command argument for `npm install` : `--brn-binary-site=https://mysite.com/downloads`
```

Comment:
It looks like the `--brn...` is supposed to be an argument to `npm install`, but I'm not sure why these are separated as if two different commands and what the colon is supposed to mean. What am I missing?

Reply:
It's just me trying to separate the command and the parameter. it actually makes more sense to join both.